capability/capability_test.go

```diff
@@ -24,47 +26,43 @@ const (
 func TestLastCap(t *testing.T) {
 	last, err := LastCap()
 	switch runtime.GOOS {
-	case "linux":
-		if err != nil {
-			t.Fatal(err)
-		}
 	default:
 		if err == nil {
 			t.Fatal(runtime.GOOS, ": want error, got nil")
 		}
 		return
+	case "linux":
```

Comment:
I like it that you can have `default` as the first case in switch; it improves readability in here (otherwise I would have to add a comment saying "Below is linux-only stuff").